apps/mobile/app/(import-accounts)/form.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { ImportAccountFormContainer } from '@/src/features/ImportReadOnly'
+function ImportAccountFormScreen() {
```

Comment:
can we add a new line between the component declaration and import?

Reply:
We can, but my view is that if lint is passing, then it is ok. We have to add a lint rule that looks at this :)